server/data/data-jmap/src/test/java/org/apache/james/jmap/api/projections/MessageFastViewProjectionHealthCheckTest.java

```diff
@@ -0,0 +1,220 @@
+package org.apache.james.jmap.api.projections;
+
+import static org.apache.james.jmap.api.projections.MessageFastViewProjectionContract.MESSAGE_FAST_VIEW_PRECOMPUTED_PROPERTIES_1;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.stream.IntStream;
+
+import org.apache.james.core.healthcheck.ComponentName;
+import org.apache.james.core.healthcheck.Result;
+import org.apache.james.jmap.memory.projections.MemoryMessageFastViewProjection;
+import org.apache.james.mailbox.model.TestMessageId;
+import org.apache.james.metrics.tests.RecordingMetricFactory;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import reactor.core.publisher.Mono;
+
+class MessageFastViewProjectionHealthCheckTest {
+
+    private static final ComponentName COMPONENT_NAME = new ComponentName("MessageFastViewProjection");
+
+    private static final TestMessageId MESSAGE_ID_1 = TestMessageId.of(1);
+
+    private MessageFastViewProjectionHealthCheck testee;
+    private MemoryMessageFastViewProjection fastViewProjection;
+
+    @BeforeEach
+    void setUp() {
+        RecordingMetricFactory metricFactory = new RecordingMetricFactory();
+        fastViewProjection = new MemoryMessageFastViewProjection(metricFactory);
```

Comment:
you don't really need this projection, you can write the tests with only the metricFactory